### DISCUSSION TITLE
Episode 120: C++ Safety, Tuples & Variants with Zach Laine! (Part 4)

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#11] [Episode]
Episode Link: https://adspthepodcast.com/2023/03/10/Episode-120.html

Feel free to write comments or ask questions 🙂

**Comment by VincentZalzal** [comment]
I also find it hilarious when I hear Bryce sped up at 10x, please continue :)

**Reply by codereport** [reply]
Haha, thanks for the feedback! Will continue to do so : )
